src/pages/workspaces/workspace/DeleteWorkspaceModal.js

```diff
@@ -55,6 +59,15 @@ export default class DeleteWorkspaceModal extends Component {
           marginTop: '1rem'
         }
       }, 'This cannot be undone.'),
+      div({ style: { marginTop: '1rem' } }, [
+        label({ htmlFor: 'understand' }, ['Please type \'I Understand\' to continue:']),
```

Comment:
I can imagine someone typing "I understand" without realizing what they're agreeing to. Perhaps we should have them type "delete workspace"?

Reply:
Sounds good. Will change.